### ISSUE TITLE
Large ctags files will crash atom

### ISSUE THREAD

**Issue by forivall** [yongkangchen/atom-ctags#77]
I built ctags for the linux kernel source, and it obviously crashed atom when I tried to load it. The ctags file itself is 380MB.

My suggestion would be to use the node `ctags` module (which is what symbols uses), since it uses the native c library to memmap the ctags file etc.

**Comment by yongkangchen** [comment]
see README:
```
'extraTagFiles' setting, add specified tagFiles.(Make sure you tag file generate with --fields=+KSn)
```

**Comment by forivall** [comment]
That's exactly what I did, I created an extraTagFile of the linux kernel source tree that was 380MB, and atom crashed when your plugin tried to load it, because you're doing it in pure javascript instead of the node bindings to the ctags library.

**Comment by yongkangchen** [comment]
Fixed in fc7f2b0e88d140865d199cf972c4fca3369e14c2

**Comment by yongkangchen** [comment]
Will be published after release of https://github.com/atom/node-ctags/pull/9
